stdlib/3.4/asyncio/queues.pyi

```diff
@@ -33,10 +33,9 @@ class Queue(Generic[_T]):
     @coroutine
     def get(self) -> Generator[Any, None, _T]: ...
     def get_nowait(self) -> _T: ...
-    if sys.version_info >= (3, 4):
```

Comment:
I'm really curious why this test was there to begin with -- asyncio doesn't support versions before 3.4, and this file is in the 3.4 directory.